javascript/javascript1/week3/homework5/noteTaking.js

```diff
@@ -0,0 +1,43 @@
+let notes = [];
+function saveNote (content, id){
+    const newObj = {"content": content, "id": id};
+    return notes.push(newObj)
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log(notes); 
+
+function getNote(id){
+ for (let i = 0; i< notes.length; i++){
+    if(notes[i].id === id) {
+        return notes[i]
+    }
+ }
+ return alert('Invalid id, try again with an existing id value')
+}
+
+const firstNote = getNote(1);
+console.log(firstNote);
+
+function logOutNotesFormatted() {
+    for (let note of notes) {
+        console.log(`The note with id: ${note.id}, has the following note text: ${note.content} `)
+    }
+  }
+  
+  logOutNotesFormatted();
+
+  function deleteNote(id){
+    for (let i = 0; i< notes.length; i++){
+        if(notes[i].id === id) {
+         const showNote = console.log(`the note with id ${id} and following content: ${notes[i].content} / has been deleted`);
+         notes.splice(notes[i], 1)
+         return showNote;
+        }
+     }
+     return alert('Invalid id, try again with an existing id value')
+  }
+
+  deleteNote(2)
```

Comment:
If you log the notes after the deletion, do you see what you expect? 🐛 It took me a second to spot the troublesome code; can you spot it? 😄 

Another nitpick is that on line 35, `showNote` is assigned the value of `console.log(...)`, is this necessary? 🔍